HW_7/task_34.py

```diff
@@ -0,0 +1,17 @@
+input_text = input("Write your song sounds: ")
+check_letters = list('аеёиоуыэюя')
```

Comment:
Это было не обязательно
```suggestion
check_letters = 'аеёиоуыэюя'
```